### ISSUE TITLE
⌨ Lists direct child must be `li`

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#9672]
## ⌨ Accessibility Issue

Our new lists of skill levels on the pool advertisement page have parentheses as direct children but should be `li`.

### WCAG 2.1 Criteria

- [1.3.1: Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
- [Technique H48: Using ol, ul and dl for lists or groups of links](https://www.w3.org/WAI/WCAG21/Techniques/html/H48.html)
- [ul and ol must only directly contain li, script or template elements](https://dequeuniversity.com/rules/axe/4.8/list)

## 🦋 Expected Behaviour

the `ul` element should only have `li` as the direct child.

## 📋 Steps to Reproduce

1. Navigate to a pool advertisement
2. Open then inspector targeting the list of skill levels on the skill accordions
3. Confirm that parentheses exist as the first and last child of the list

## 📸 Screenshot

![Screenshot 2024-03-07 080805](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/d9b83957-86b3-4333-8f0d-7a1d6f6b171e)

## 🙋‍♀️ Proposed Solution

1. This does not seem to be an actual list, we could just remove the `ul` and `li` elements and use `span` instead
2. The parentheses dont't seem to serve a purpose so we could remove them
3. If we need them there and there is a purpose, they must be moved outside of the `ul` element

## ✅ Acceptance Criteria

- [x] Option 1: `ul` and `li` elements have been replaced with `span`
- [x] Option 2: Parentheses have been removed
- [ ] Option 3: Parentheses moved outside of the `ul` element

**Comment by esizer** [comment]
Actually looks like a ticket was created to remove them: #9657

I still feel like maybe this is not really a list and should use `span` instead but it is lower priority now if we plan on removing the parentheses.

**Comment by tristan-orourke** [comment]
Remove the parenthesis AND the ul/li (replace with spans)
